README.md

```diff
@@ -184,6 +184,13 @@ On MacOS
 conda install pkg-config libuv
 ```
 
+On Windows
+```bash
+# Add these packages and set libuv_ROOT environment variable if torch.distributed is needed
```

Comment:
Let's add one line:

```
# Distributed package support on Windows is a prototype feature and is subject to changes. 
```

Reply:
@jspisak @gchanan is it OK to mention a prototype feature in master `README.md`?